pkgs/development/python-modules/karton-core/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, minio
+, redis
+}:
+
+buildPythonPackage rec {
+  pname = "karton-core";
+  version = "4.2.0";
+
+  src = fetchFromGitHub {
+    owner = "CERT-Polska";
+    repo = "karton";
+    rev = "v${version}";
+    sha256 = "08j1bm9g58576sswcrpfczaki24nlqqaypp7qv1rxxwsyp5pq6h6";
+  };
+
+  propagatedBuildInputs = [ minio redis ];
+
+  checkPhase = "python -m unittest discover";
+
+  meta = with lib; {
+    description = "Distributed malware processing framework based on Python, Redis and MinIO";
+    homepage = "https://github.com/CERT-Polska/karton";
```

One can get to the documentation from the source repo with one click but from my point of view it's nicer to link directly to the documentation.

```suggestion
    homepage = "https://karton-core.readthedocs.io/";
```